app/javascript/src/components/Dashboard/Contacts/AddContactForm.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form } from "formik";
+import { Input, Select } from "neetoui/formik";
+import { Button, Label } from "neetoui/v2";
+
+import FORM_INITIAL_VALUES from "constants/formInitialValues";
+import FORM_SELECT_OPTIONS from "constants/formSelectOptions";
+import FORM_VALIDATION_SCHEMAS from "constants/formValidationSchemas";
```

Comment:
Please refer to my previous comment regarding placing constants in the scope of a component.